### ISSUE TITLE
add deepinfra provider

### ISSUE THREAD

**Issue by morandalex** [continuedev/continue#725]
### Validations

- [ ] I believe this is a way to improve. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [ ] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement) that requests the same enhancement

### Problem

I cannot add deep infra provider. seems not working 

```
  "models": [
    {
      "title": "deepinfra-mixtralai",
      "provider": "openai",
      "model": "mistralai/Mixtral-8x7B-Instruct-v0.1",
      "apiKey": "<apikey>",
      "apiBase": "https://api.deepinfra.com/v1/openai"
    },
]
```

ollama works, so I would say is a problem coming from continue.dev

**Comment by sestinj** [comment]
As per [docs here](https://deepinfra.com/docs/advanced/openai_api) they only support this OpenAI API for a few models, mixtral not included. So you're right, we'll have to add a new provider. Will do this soon and let you know when the update is ready

**Comment by sestinj** [comment]
Added in [this commit](https://github.com/continuedev/continue/commit/1b397e7600f05aa6d9a03d529a9c6adff5308234) and will be available in ~20min in pre-release version 0.9.1

**Comment by morandalex** [comment]
awesome guys!
